src/views/DetailedView/DescriptionContainer/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { FunctionComponent } from "react";
+import { VenueDetails } from "../../../shared/types/VenueDetails.ts";
+import { AmenitiesList } from "./AmenitiesList";
+import { StayingDetails } from "./StayingDetails";
+import {
+  DescriptionPaper,
+  DescriptionDivider,
+} from "./DescriptionContainer.styled.tsx";
+
+interface DescriptionContainer {
+  venueDetails: VenueDetails;
+}
+export const DescriptionContainer: FunctionComponent<DescriptionContainer> = ({
+  venueDetails,
+}) => {
+  const amenitiesString =
+    venueDetails.features.generalAmenities.join() +
+    "," +
+    venueDetails.features.roomAmenities.join() +
+    "," +
+    venueDetails.features.handicapAccesibility.join() +
+    "," +
+    venueDetails.features.neighbourhoods.join();
+  const amenitiesArray = amenitiesString.split(",");
```

Comment:
I think you can use spread operator here
```suggestion
  const amenities = [
    ...venueDetails.features.generalAmenities,
    ...venueDetails.features.roomAmenities,
    ...venueDetails.features.handicapAccesibility,
    ...venueDetails.features.neighbourhoods
  ];
```